container/container.go

```diff
@@ -58,136 +83,169 @@ func New(opts ...Option) *Container {
 	return cnr
 }
 
-// ToV2 returns the v2 Container message.
-//
-// Nil Container converts to nil.
-func (c *Container) ToV2() *container.Container {
+// CalculateID calculates container identifier
+// based on its structure.
+func CalculateID(c *Container) *cid.ID {
 	if c == nil {
 		return nil
 	}
 
-	return &c.v2
-}
-
-// NewVerifiedFromV2 constructs Container from NeoFS API V2 Container message.
-//
-// Does not perform if message meets NeoFS API V2 specification. To do this
-// use NewVerifiedFromV2 constructor.
-func NewContainerFromV2(c *container.Container) *Container {
-	cnr := new(Container)
-
-	if c != nil {
-		cnr.v2 = *c
-	}
-
-	return cnr
-}
+	var v2 container.Container
+	c.WriteToV2(&v2)
 
-// CalculateID calculates container identifier
-// based on its structure.
-func CalculateID(c *Container) *cid.ID {
-	data, err := c.ToV2().StableMarshal(nil)
+	data, err := v2.StableMarshal(nil)
 	if err != nil {
 		panic(err)
 	}
 
-	id := cid.New()
+	var id cid.ID
 	id.SetSHA256(sha256.Sum256(data))
 
-	return id
+	return &id
 }
 
-func (c *Container) Version() *version.Version {
+// Version returns container's version.
```

Comment:
This statement can be interpreted as a revision of a specific container.
```suggestion
// Version returns version of the NeoFS API protocol by which the container was created.
```